geonode/client/hooksets.py

```diff
@@ -73,9 +73,6 @@ def map_download_template(self, context=None):
 
     # Map Persisting
     def viewer_json(self, conf, context=None):
-        if not context:

```

Comment:
Can we please change this to

`def viewer_json(self, conf, context={}):`

Reply:
Sure, it seems a better default.